vic/vic_run/src/surface_fluxes.c

```diff
@@ -261,6 +261,7 @@ surface_fluxes(bool                 overstory,
     coverage = snow->coverage;
     snow_energy = (*energy);
     soil_energy = (*energy);
+    iter_soil_energy = (*energy);
```

Comment:
@bartnijssen: yes, the intel compilers are returning this warning: 

```
../../vic_run/src/surface_fluxes.c(512): warning #3656: variable "iter_soil_energy" may be used before its value is set
                              (last_snow_flux + iter_soil_energy.snow_flux) / 2.;
```

What I've done here is set the initial value of `iter_soil_energy` to the `energy` struct prior to the begining of the iteration. My reading of the code tells me VIC should never reach line 512 prior to the setting of `iter_soil_energy` but the compiler can't see that and I could use a second set of eyes to make sure this isn't going to change the answers in a corner case.